File: app/src/main/java/avanger/co/id/PanduanPengguna.java
package avanger.co.id;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class PanduanPengguna extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_panduan_pengguna);

        Button btnKembaliMenu = findViewById(R.id.KembaliMenu);

        btnKembaliMenu.setOnClickListener((v) -> finish());
    }
}

File: app/src/main/java/avanger/co/id/PanduanPengguna.kt
package avanger.co.id

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import avanger.co.id.databinding.ActivityPanduanPenggunaBinding

class PanduanPengguna : AppCompatActivity() {
    private lateinit var binding: ActivityPanduanPenggunaBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityPanduanPenggunaBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.KembaliMenu.setOnClickListener { finish() }
    }
}